docs/validators/label.rst

```diff
@@ -23,6 +23,42 @@ Label
          message: 'Come message...'
       # all of the message sub-option is optional
 
+::
+
+    - do: label
+      and:
+        - must_include:
+            regex: 'DO NOT MERGE'
```

Comment:
```suggestion
            regex: 'big|medium|small'
```

Reply:
I like this suggestion. It's a better more realistic example.